services/device-registry/src/main/java/org/eclipse/hono/deviceregistry/FileBasedCredentialsService.java

```diff
@@ -592,16 +580,6 @@ protected void verifyBcryptPasswordHash(final String pwdHash) {
         }
     }
 
-    private void checkHashedPassword(final PasswordSecret secret) {
```

Comment:
Was this intentional? It doesn't seems related to the rest of the changes.

Reply:
It caused a warning, since the method was not used by anyone..